src/app/pages/confirm-test-details/confirm-test-details.page.ts

```diff
@@ -13,6 +13,7 @@ export class ConfirmTestDetailsPage implements OnInit {
   constructor(
     private navController: NavController,
```

Comment:
can u remove navcontroller if your not using it any more

Reply:
not really part of this merge but will do. will be lots of skeleton pages suing it atm they will be over-written as migrated